Marlin/src/module/motion.h

```diff
@@ -217,7 +217,7 @@ void set_axis_is_not_at_home(const AxisEnum axis);
 void homeaxis(const AxisEnum axis);
 
 #if ENABLED(SENSORLESS_HOMING)
-  void sensorless_homing_per_axis(const AxisEnum axis, const bool enable=true);
+  void sensorless_homing_per_axis(const AxisEnum axis, const bool enable);
```

Comment:
The definition for this function is removed, so should this be removed?. But it is still being called by this:
```cpp
inline void delta_sensorless_homing(const bool on=true) {
  sensorless_homing_per_axis(A_AXIS, on);
  sensorless_homing_per_axis(B_AXIS, on);
  sensorless_homing_per_axis(C_AXIS, on);
}
```

Reply:
Just changed but not removed. I'll go based on the assumption that `A_AXIS` corresponds with `stepperX` and so on.

Reply:
The `sensorless_homing_per_axis` function is not present in `motion.cpp`. It was replaced by `start_sensorless_homing_per_axis` and `end_sensorless_homing_per_axis`.